examples/uiCircleCaptureExample.jl

```diff
@@ -29,18 +28,29 @@ attachEventSystem(renderer, mouseState, keyboardState)
 
 function runApp(renderer)
 	init(renderer)
-    #render(renderer)
-	object = mouseState.leftClick ? renderer.scene.objects[2] : renderer.scene.objects[1]
-	WGPUgfx.render(renderer.renderPass, renderer.renderPassOptions, object, renderer.scene.cameraId)
+	render(renderer)
 	deinit(renderer)
 end
 
 main = () -> begin
```

Comment:
This main function was supposed to be hidden on user code. `runApp` is where logic should go. Just letting you know my approach and original thoughts. Let me know if this is not convenient.